modular_zubbers/code/datums/emotes.dm

```diff
@@ -0,0 +1,180 @@
+	cooldown = 7.5 SECONDS
+
+/datum/emote/living/audio/wtune
+	key = "whistle_tune"
+	key_third_person = "whistle_tunes"
+	message = "whistles a tune."
+	message_mime = "makes an expression as if whistling."
+	vary = TRUE
+	sound = 'modular_zubbers/sound/voice/wtune1.ogg'
+
+/datum/emote/living/audio/wtune/run_emote(mob/user, params)
+	// Set random emote sound
+	sound = pick('modular_zubbers/sound/voice/wtune1.ogg', 'modular_zubbers/sound/voice/wtune2.ogg')
+
+	// Return normally
+	. = ..()
+
+/datum/emote/living/audio/chill
+	key = "chill"
+	key_third_person = "chills"
+	message = "feels a chill running down their spine..."
+	message_mime = "acts out a chill running down their spine..."
+	vary = TRUE
+	sound = 'modular_zubbers/sound/voice/waterphone.ogg'
+
+/datum/emote/living/audio/weh2
+	key = "weh2"
+	key_third_person = "wehs"
+	message = "let out a weh!"
+	message_mime = "acts out a weh!"
+	vary = TRUE
+	sound = 'modular_zubbers/sound/voice/weh2.ogg'
+
+/datum/emote/living/audio/weh2/run_emote(mob/user, params)
+	// Set random emote sound
+	sound = pick('modular_zubbers/sound/voice/weh2.ogg', 'modular_zubbers/sound/voice/weh3.ogg', 'modular_zubbers/sound/voice/weh_s.ogg', 'modular_zubbers/sound/voice/waa.ogg')
+
+	// Return normally
+	. = ..()
+
+/datum/emote/living/audio/ara_ara
+	key = "ara"
+	key_third_person = "aras"
+	message = "coos with sultry surprise~..."
+	message_mime = "exudes a sultry aura~"
+	vary = TRUE
+	sound = 'modular_zubbers/sound/voice/ara-ara.ogg'
+
+/datum/emote/living/audio/ara_ara/run_emote(mob/user, params)
+	// Set random emote sound
+	sound = pick('modular_zubbers/sound/voice/ara-ara.ogg', 'modular_zubbers/sound/voice/ara-ara2.ogg')
+
+	// Return normally
+	. = ..()
+
+/datum/emote/living/audio/nya2
+	key = "nya2"
+	key_third_person = "nya2s."
+	message = "Lets out a nya~."
+	message_mime = "Nyas silently~"
+	vary = TRUE
+	sound = 'modular_zubbers/sound/voice/felinid/nya-1.ogg'
+
+/datum/emote/living/audio/felinid/nyaha/run_emote(mob/user, params)
+	sound = pick('modular_zubbers/sound/voice/felinid/nya-1.ogg', 'modular_zubbers/sound/voice/felinid/nya-2.ogg')
+
+/datum/emote/living/audio/teshari/teshcough
+	key = "teshcough"
+	key_third_person = "teshcoughs"
+	message = "coughs cutely!"
+	message_mime = "acts out a cute cough!"
+	vary = TRUE
+	sound = 'modular_zubbers/sound/voice/teshari/tesharicougha.ogg'
+
+/datum/emote/living/audio/teshari/cought/run_emote(mob/user, params)
+	sound = pick('modular_zubbers/sound/voice/teshari/tesharicoughb.ogg', 'modular_zubbers/sound/voice/teshari/tesharicougha.ogg')
+
+/datum/emote/living/audio/teshari/scream
+	key = "teshscream"
+	key_third_person = "teshscreams"
+	message = "screams like a chicken!"
+	message_mime = "acts out a chicken scream."
+	vary = TRUE
+	sound = 'modular_zubbers/sound/voice/teshari/teshariscream.ogg'
+
+/datum/emote/living/audio/teshari/sneeze
+	key = "teshsneeze"
+	key_third_person = "teshsneeze"
+	message = "sneezes cutely."
```

Comment:
I need a Teshari expert on this- how the heck do these little buggers sneeze "cutely" @projectkepler-ru

Reply:
I dunno if I'd consider Kepler a "Teshari expert"